extensions/adobe/experience/campaign/experienceevent-all.schema.json

```diff
@@ -244,8 +244,8 @@
           "properties": {
             "xdm:eventID": {
               "title": "Event ID",
-              "type": "string",
-              "description": "ID that identifies the event sent by the data source. This is used by Campaign orchestration to identify the event without inspecting its payload to determine which action should be triggered when the event is received. The value is automatically generated in Campaign at event creation time, it is globally unique across organizations."
+              "description": "ID that identifies the event sent by the data source. This is used by Campaign orchestration to identify the event without inspecting its payload to determine which action should be triggered when the event is received. The value is automatically generated in Campaign at event creation time, it is globally unique across organizations.",
+              "$ref": "https://ns.adobe.com/experience/campaign/orchestration/eventid"
```

Comment:
Hi @reivaxy, we need to change this in order to avoid a double-nesting of the `xdm:eventID` property.  The best way to do this is simply to re-define the eventID right here, rather than reference it.  This will compose as expected.

Reply:
Thanks @kstreeter, I did it, please check it's good now.